core/src/main/java/com/klaytn/caver/utils/AccountKeyPublicUtils.java

```diff
@@ -55,4 +65,158 @@ public static AccountKeyPublic decompressKey(String compressedPublicKey) {
                 Numeric.toHexStringWithPrefixZeroPadded(ecPoint.getAffineYCoord().toBigInteger(), 64)
         );
     }
+
+    /**
+     * Creates ECPoint instance using compressed public key
+     * @param compressedPublicKey compressed public key
+     * @return ECPoint
+     */
+    public static ECPoint getECPoint(String compressedPublicKey) {
```

Comment:
I think It would be nice if uncompressedPublicKey can also use this function to get ECPoint without getting x and y points.

Reply:
OK, I'll modify it.